### ISSUE TITLE
Missing icons

### ISSUE THREAD

**Issue by cevap** [cevap/ion#7]
Qt's icon is now showing, icon has to be recreated it misses some parts:

![image](https://user-images.githubusercontent.com/24996551/38103474-b06c2a06-3386-11e8-940e-0088481275ac.png)

Requires icons for all networks:

-  [x] MAIN (c46bd548b3662829d34f531a149fb9b76208b2eb)
   - [x] New splash image: [src/qt/res/images/splash.png](https://github.com/cevap/ion/blob/master/src/qt/res/images/splash.png)
   ![splash](https://user-images.githubusercontent.com/24996551/38137143-43d34c06-3423-11e8-9078-474628b3c340.png)
-  [x] TESTNET (12cb98c0afd7f685e1e7f45f7bc6ce7a17f40aa1)
   - [x] New splash image: [src/qt/res/images/splash_testnet.png](https://github.com/cevap/ion/blob/master/src/qt/res/images/splash_testnet.png)
   ![splash_testnet](https://user-images.githubusercontent.com/24996551/38137127-2aaa135e-3423-11e8-9d2e-eadbfe114991.png)
-  [x] REGTEST (12cb98c0afd7f685e1e7f45f7bc6ce7a17f40aa1)
   - [x] New splash image: [src/qt/res/images/splash_regtest.png](https://github.com/cevap/ion/blob/master/src/qt/res/images/splash_regtest.png)
   ![splash_regtest](https://user-images.githubusercontent.com/24996551/38137014-6c995352-3422-11e8-8619-9b14a4d81c7d.png)
-  [x] unittest (12cb98c0afd7f685e1e7f45f7bc6ce7a17f40aa1)
   - [x] New splash image: [src/qt/res/images/splash_unittest.png](https://github.com/cevap/ion/blob/master/src/qt/res/images/splash_unittest.png)
   ![splash_unittest](https://user-images.githubusercontent.com/24996551/38137226-bced2616-3423-11e8-8672-678595c817c3.png)

**Comment by cevap** [comment]
main should be fixed with this commit: c46bd548b3662829d34f531a149fb9b76208b2eb

**Comment by cevap** [comment]
please confirm if on other systems icons are missing or not

**Comment by cevap** [comment]
Additional commit which applies here too: e36f97ab3a66d125e3fc2195affcdb2f9e0a1a96

**Comment by cevap** [comment]
can anybody check and verify on their os if all icons are present?

**Comment by FornaxA** [comment]
Re-appearance of systray icon under ubuntu/gnome is confirmed.

**Comment by FornaxA** [comment]
New splash screens are nice (almost as nice as the ones we're leaving behind :))

**Comment by cevap** [comment]
Thanks for confirmation. What about ALT+TAB?

I pushed splashscreens for each network. This should be just a refference, and all SVG are included. Here, the image has to be moved to the left few pixels, as now copyright sign is slightly unvisible and it does not look very nice, I will change this with one of next commits, prepared some other stuff. 

I also repaired icons, as they did not include the iconset, just one icon.

I wait for checks, on my mint ALT+TAB does not show any logo, but it should shoow. Simply aliases etc should be checked if they include some ION instead of ion or bitcoin instead of ion.
